packages/image_picker/image_picker/example/android/app/build.gradle

```diff
@@ -35,7 +35,7 @@ android {
     defaultConfig {
         applicationId "io.flutter.plugins.imagepicker.example"
         minSdkVersion 16
-        targetSdkVersion 28
+        targetSdkVersion 30
```

Comment:
Is it necessary upgrade to `targetSdkVersion` to API level 30? This looks like you maybe updated it for testing purposes and forgot to change it back.